Installation.md

```diff
@@ -13,8 +13,8 @@ Packages will be released nightly after automated building is set up.
 | Download | Date [(last commit)](https://github.com/apple/swift/commit/280486afdc0c940af345c614a80826fcc8326abc) |
```

Comment:
The "last commit" on this line is no longer accurate. Could you remove it?

Reply:
Done,